packages/api-rx/test/e2e/transfer.spec.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017-2018 @polkadot/api-rx authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { switchMap } from 'rxjs/operators';
+import testingPairs from '@polkadot/util-keyring/testingPairs';
+
+import Api from '../../src';
+
+const keyring = testingPairs();
+
+describe.skip('e2e transfer', () => {
+  let api;
+
+  beforeEach(async () => {
+    api = await Api.create().toPromise();
```

Comment:
could rename `Api` to `ApiRx` on line 8 and 16?

Reply:
As above.